testing/util/blob.go

```diff
@@ -36,39 +22,13 @@ func HydrateBlobSidecar(b *ethpb.DeprecatedBlobSidecar) *ethpb.DeprecatedBlobSid
 	if b.KzgProof == nil {
 		b.KzgProof = make([]byte, fieldparams.BLSPubkeyLength)
 	}
-	return b
-}
-
-// HydrateSignedBlindedBlobSidecar hydrates a signed blinded blob sidecar with correct field length sizes
-// to comply with SSZ marshalling and unmarshalling rules.
-func HydrateSignedBlindedBlobSidecar(b *ethpb.SignedBlindedBlobSidecar) *ethpb.SignedBlindedBlobSidecar {
-	if b.Signature == nil {
-		b.Signature = make([]byte, fieldparams.BLSSignatureLength)
-	}
-	if b.Message == nil {
-		b.Message = &ethpb.BlindedBlobSidecar{}
-	}
-	b.Message = HydrateBlindedBlobSidecar(b.Message)
-	return b
-}
 
-// HydrateBlindedBlobSidecar hydrates a blinded blob sidecar with correct field length sizes
-// to comply with SSZ marshalling and unmarshalling rules.
-func HydrateBlindedBlobSidecar(b *ethpb.BlindedBlobSidecar) *ethpb.BlindedBlobSidecar {
-	if b.BlockRoot == nil {
-		b.BlockRoot = make([]byte, fieldparams.RootLength)
-	}
-	if b.BlockParentRoot == nil {
-		b.BlockParentRoot = make([]byte, fieldparams.RootLength)
-	}
-	if b.KzgCommitment == nil {
-		b.KzgCommitment = make([]byte, fieldparams.BLSPubkeyLength)
-	}
-	if b.KzgProof == nil {
-		b.KzgProof = make([]byte, fieldparams.BLSPubkeyLength)
-	}
-	if b.BlobRoot == nil {
-		b.BlobRoot = make([]byte, fieldparams.RootLength)
+	if b.CommitmentInclusionProof == nil {
+		r := make([][]byte, fieldparams.KzgCommitmentInclusionProofDepth)
+		for i := range r {
+			r[i] = make([]byte, fieldparams.RootLength)
+		}
+		b.CommitmentInclusionProof = r
```

Comment:
I think Kasey wrote a `fakeEmptyProof` helper function that you may be able to reuse it here